packages/core/solidity/src/helpers.ts

```
@@ -0,0 +1,25 @@
+// Example:
+// - makeUpgradeable('AccountERC7579', false) == 'AccountERC7579'
+// - makeUpgradeable('AccountERC7579', 'uups') == 'AccountERC7579Upgradeable'
+// - makeUpgradeable('@openzeppelin/contracts/account/extensions/draft-AccountERC7579.sol', false) == '@openzeppelin/contracts/account/extensions/draft-AccountERC7579.sol'
+// - makeUpgradeable('@openzeppelin/contracts/account/extensions/draft-AccountERC7579.sol', 'uups') == '@openzeppelin/contracts-upgradeable/account/extensions/draft-AccountERC7579Upgradeable.sol'
+export function makeUpgradeable(input: string, upgradeable: CommonOptions['upgradeable'] = false): string {
+    switch (upgradeable) {
+        case false:
+            return input;
+        case 'uups':
+        case 'transparent':
+            const { dir, name, ext } = path.parse(input);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)